src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -106,15 +134,33 @@ protected override void OnPersist(ApplicationEngine engine)
             // Distribute GAS for committee
 
             var gasPerBlock = GetGasPerBlock(engine.Snapshot);
-            var committee = GetCommitteeMembers(engine.Snapshot).ToArray();
+            var committee = GetCommittee(engine.Snapshot);
             var pubkey = committee[engine.Snapshot.PersistingBlock.Index % ProtocolSettings.Default.CommitteeMembersCount];
             var account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
             GAS.Mint(engine, account, gasPerBlock * CommitteeRewardRatio / 100);
 
+            // Record the cumulative reward of the voters of each committee
+
+            var voterRewardPerCommittee = gasPerBlock * VoterRewardRatio / 100 / (ProtocolSettings.Default.CommitteeMembersCount + ProtocolSettings.Default.ValidatorsCount);
```

Comment:
Validators are included inside the Committee, they will be rewarded twice as committee and as validator?

Reply:
I think so, as dictated in the description: 
> The voters who voted for a consensus node are assigned 2/28 of this part according to the voting weight, and the voters who voted for a committee member (non-consensus node) are assigned 1/28 of this part according to the voting weight. (Implemented by this PR)

Reply:
Yes.

Reply:
It is mainly used to ensure that the validators' votes are concentrated and not too scattered.

Reply:
Correct me if I'm wrong, but we've got rounding error even here:
```
> 500000000.0*85/100/(21+7)
=> 15178571.42857143
```
Then it only gets worse with votes/neo amount.